cogs/welcome/constants.py

```diff
@@ -16,10 +16,18 @@
 KEY_WELCOME_CHANNEL_SETTINGS = "welcomeChannelSettings"
 KEY_POST_FAILED_DM = "postFailedDm"
 KEY_JOINED_USER_IDS = "joinedUserIds"
+KEY_TOGGLE_RANDOM_IMG = "toggleImg"
+KEY_WELCOME_IMG_FOLDER = "welcomeImgs"
+KEY_WELCOME_IMG_TEMPLATE = "welcome_template.png"
```

Comment:
We aren't using these as config keys, so we shouldn't declare them as `KEY_` constants.